community/openjdk7/icedtea-jdk-fix-freetype-detection.patch

```diff
@@ -0,0 +1,20 @@
+--- openjdk/jdk/make/tools/freetypecheck/freetypecheck.c.orig
```

Comment:
I think the first two lines are wrong. Shouldn't it be different in prefix, not suffix?
e.g.:
```
--- old/openjdk/jdk/make/tools/freetypecheck/freetypecheck.c
+++ new/openjdk/jdk/make/tools/freetypecheck/freetypecheck.c
```